### PR TITLE
Feature toast

### PR DESCRIPTION
**Pull request by abhijitdotsharma**
Toast Component with dismiss icon
Temp-toast-img

### CODE REVIEW

**Comment by ankit9015** [comment]
Suggestion, since it's a toast, a small video will be more helpful than an image to showcase your component. 
You can use fonepaw screen recorder, this is what I am using. https://www.fonepaw.com/screen-recorder/

**Comment by sushil-mod** [comment]
code looks nice ,you can merge!

**Review by Raghav888 — APPROVED** [review]

File: components/component.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+:root{
+    --color-primary : #FF3B30;
+    --color-secondary: grey;
+    --color-success: green;
+    --color-danger: red;
+    --color-warning: rgb(218, 218, 54);
+    --color-info: rgb(147, 147, 238);
+    --color-white: #Fff6ea;
+    --color-black: #282828;
```
Comment (ozairkhan2810) [inline components/component.css:11]: use all in one form like names or hex codes or rgb

**Review by ozairkhan2810 — APPROVED** [review]
just one suggestion, rest looks good to merge

File: components/component.css (hunk 1 above)
Comment (abhijitdotsharma) [reply components/component.css:11]: I haven't finalized the colors yet, when I do I'll keep your suggestion in mind. Thanks

**Pull request merged**